src/settings/index.js

```diff
@@ -83,13 +85,22 @@ class ResourceSharingSettings extends React.Component {
       perm: page.perm,
     }));
 
-    const dynamic = sections.map(section => ({
-      route: section,
-      label: intl.formatMessage({ id: `ui-rs.settingsSection.${section}` }),
-      component: (props) => <SettingPage sectionName={section} {...props} />,
-    }));
+    const dynamic = sections.map(section => {
+      const sectionFormatted = snakeToCamel(section);
+      return (
+        {
+          route: sectionFormatted,
+          label: intl.formatMessage({ id: `ui-rs.settingsSection.${sectionFormatted}` }),
+          component: (props) => <SettingPage sectionName={section} {...props} />,
+        }
+      );
+    });
 
-    return persistent.concat(dynamic).sort(sortByLabelCaseInsensitive);
+    console.log("PERSISTENT PAGES: %o", persistent)
```

Comment:
Maybe take out these `console.log`s?

Reply:
They have been :) I put up the PR, and then before I realised my mistake Ian had merged it. Separate commit later takes them out